docs/Upgrades.md

```diff
@@ -1,4 +1,4 @@
-# Proxies
+# Upgrades
```

Comment:
Why this change? The way I thought of this is that proxying is a broader category within which upgrades live

Reply:
The linter doesn't like duplicate headings. I agree with `proxy` as the broader category. I'll change with inline HTML and use a separate id so we don't get the error